examples/seismic/model.py

```diff
@@ -461,3 +462,33 @@ def vp(self, vp):
             initialize_function(self.m, 1 / (self.vp * self.vp), self.nbpml)
         else:
             self.m.data = 1 / vp**2
+
+    def __getstate__(self):
+        state = dict(self.__dict__)
+        state.pop('grid')
```

Comment:
When using pop I explicitly want to remove this key:value pair from the state object. This surgery is required to prevent more information being dumped or communicated via pickling that necessary.

Reply:
is that what they claim to do in the documentation that you're referring to? Having a method called ``*get*`` that actually pops stuff? 

I think it's worth adding a comment

Reply:
I'm dumb, or perhaps I was drunk. I thought you're calling pop on self. Don't know why. Ignore all this please.